config.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
 	port: 4200, //- Port of the API and Dashboard
 	scopes: ["identify", "guilds", "applications.commands"], //- Discord OAuth2 Scopes
 	serverDeafen: true, //- If you want bot to stay deafened
-	defaultVolume: 100, //- Sets the default volume of the bot, You can change this number anywhere from 1 to 100
+	defaultVolume: 100, //- Sets the default volume of the bot, You can change this number anywhere from 1 to 9007199254740991 (JS Integer limit)
+	maxVolume: 125, //- Sets the maximum volume of the bot, You can change this number anywhere from 1 to 9007199254740991 (JS Integer limit)
```

Comment:
I also disagree on max volume configuration its confusing and not really useful. Maybe you can add a in the command description volume can be between 1 < volume < 200. Instead of 125 which I think is just an arbitrary number. Also why does the comment now says the number can be 9007199254740991?

Reply:
the number is 9007199254740991 because Darren commited it in the master branch. so I thought I'd use that.
https://github.com/SudhanPlayz/Discord-MusicBot/blob/7a25a1e66d020ef853277aafbb8d26d5dfa36ade/botconfig.js#L12

Reply:
I'll change it to 200